### PR TITLE
Add delete event functionality

### PR DESCRIPTION
**Pull request by chibujax**
#### What Does This PR Do?
- Add delete event functionality

#### How can this be manually tested?
- Navigate to event you own, then click the delete event button
#### What are the relevant pivotal tracker stories?
- #161673266

### CODE REVIEW

File: client/pages/Event/EventDetailsPage.jsx [hunk 1]
```diff
@@ -259,9 +270,14 @@ EventDetailsPage.defaultProps = {
   event: [],
   events: [],
   activeUser: { id: '' },
+  history: { push: '' },
   getEventAction: () => null,
+  deactivateEventAction: () => null,
 };
-const mapDispatchToProps = dispatch => bindActionCreators({ getEventAction: getEvent }, dispatch);
+const mapDispatchToProps = dispatch => bindActionCreators({
```
Comment (kenneth-kip) [inline client/pages/Event/EventDetailsPage.jsx]: do you mind adding a space before line 270 and after line 280 ?

**Comment by kenneth-kip** [comment]
I know this is not from your PR. When I go to the events details page. 
image
If its small fix, help us out here or if its a bit trciky let me know.

**Comment by kenneth-kip** [comment]
Otherwise works fine.

**Comment by chibujax** [comment]
> Otherwise works fine.

Its actually from my PR. Fixed.

**Pull request merged**